src/views/Profile/AdminTable/AdminTableBody.tsx

```diff
@@ -68,9 +69,11 @@ const AdminTableBody: React.FC<IAdminTableBodyProps> = ({ isAdmin }) => {
         <TableCell>{id}</TableCell>
         <TableCell className={classes.titleCol}>
           <Chip label={typeName} className={classes[chipClass] as string} />
-          <Typography variant="h6" component="p" className={classes.title}>
-            {title}
-          </Typography>
+          <Link to={`/edit-post?id=${id}`} className={classes.title}>
```

Comment:
we cannot use this here, but just as knowledge sharing: there is `generatePath` helper in `react-router-dom`

so if `/edit-post/:id` was a named route(which it is NOT and `id` parameter in query part instead of string - but let's imagine) then we could:

```
import { EDIT_POST_ROUTE } from '../...../routes.js';

  <Link to={generatePath(EDIT_POST_ROUTE, { postId: id })}

```

but here we cannot use that trick, so just saying